controllers/dependencies.go

```diff
@@ -72,34 +73,79 @@ func (r *OperatorPipelineReconciler) reconcilePipelineDependencies(meta metav1.O
 			return err
 		}
 	}
+
 	return nil
 }
 
-func (r *OperatorPipelineReconciler) removePipelineDependencyFiles(filePath string) error {
-	if err := os.RemoveAll(filePath); err != nil {
-		log.Error(err, "Couldn't remove operator-pipelines directory")
+func (r *OperatorPipelineReconciler) applyPipelineManifests(fileName string, meta metav1.ObjectMeta, pipeline *tekton.Pipeline) error {
```

Comment:
A quick question on the method signature: can we remove the third arg from both method signature and declare the var in the method body?

Reply:
Yeah. That looks like a refactoring tool gone wrong. They're unused outside of the methods, so there is no need to pass them in, nor to return them.

Reply:
This method was 3 params to begin with but just took in OBJ, so the concrete type was defined outside the method. I agree that it makes more sense to drop the 3rd param and instantiate in the method.